scalafix-core/shared/src/main/scala/scalafix/internal/patch/EscapeHatch.scala

```diff
@@ -33,49 +33,49 @@ class EscapeHatch private (
     val usedEscapes = mutable.Set.empty[EscapeFilter]
     val lintMessages = List.newBuilder[LintMessage]
 
-    def isDisabledByEscape(name: RuleName, start: Int): Boolean =
-      // annotatedEscapes takes precedence over anchoredEscapes
-      annotatedEscapes.isEnabled(name, start) match {
-        case (false, Some(culprit)) => // if disabled, there must be a escape
-          usedEscapes += culprit
-          true
-        case _ =>
-          val (enabled, culprit) = anchoredEscapes.isEnabled(name, start)
-          culprit.foreach(escape => usedEscapes += escape)
-          !enabled
-      }
+    def isDisabled(name: RuleName, pos: Position): Boolean = {
```

Comment:
I'm not ready to review this file :D So I welcome @olafurpg here.

Reply:
@MasseGuillaume is in fact the author of this file 😜